File: ejercicios/c++/ejercicio067.cpp
#include <iostream>
using namespace std;

int producto_recursivo(int numero, int multiplicador){
	if(multiplicador == 0)
		return 0;
	else if(multiplicador == 1)
		return numero;
	else
		return numero + producto_recursivo(numero, multiplicador - 1);
}


int main() {
	int numero, multiplicador;
	cout << "ingrese un numero entero : ";
	cin >> numero;
	cout << "ingrese un numero entero : ";
	cin >> multiplicador;
	if (multiplicador >= 0)
		cout << numero << " * " << multiplicador << " = " << producto_recursivo(numero, multiplicador) << endl;
	else
		cout << "El segundo numero debe ser positivo." << endl;
	return 0;
}

File: ejercicios/python/ejercicio067.py
def producto_recursivo(numero, multiplicador):
   if (multiplicador == 0):
      return 0
   if (multiplicador == 1):
      return numero
   else:
      return numero + producto_recursivo(numero, multiplicador - 1)

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   multiplicador = int(input("ingrese otro numero : "))
   if (multiplicador >= 0):
      print (str(numero) + " * " + str(multiplicador) + " = " + str(producto_recursivo(numero, multiplicador)))
   else:
      print("El segundo numero debe ser positivo.")
      
